docs/unattend.xml

```diff
@@ -16,18 +16,14 @@
          <CEIPEnabled>0</CEIPEnabled>
       </component>
       <component xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="Microsoft-Windows-Shell-Setup" processorArchitecture="amd64" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="nonSxS">
-         <ComputerName>windows-host</ComputerName>
+         <ComputerName>winhost</ComputerName>
       </component>
    </settings>
    <settings pass="oobeSystem">
       <component xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="Microsoft-Windows-Shell-Setup" processorArchitecture="amd64" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="nonSxS">
          <AutoLogon>
-            <Password>
-               <Value>MyPassword</Value>
-               <PlainText>true</PlainText>
-            </Password>
-            <Enabled>true</Enabled>
-            <Username>Administrator</Username>
+            <!-- Disable automatic logon. This is the default value. -->
+            <Enabled>false</Enabled>
```

Comment:
If this is the default, then why do we need this?

Reply:
It's safer to be explicit in this particular case.

Reply:
Sure but why?

Reply:
Given the number of issues/escalations received regarding the golden image creating process and, in particular, the security issue of having an open terminal with Administrator privileges at boot; led to make this default visible. 

It leaves an explicit record that the decision is being made, otherwise there will be no record the in the VCS, rather than the commit message, which is not exposed to the users consuming this file from the Openshift docs.

In addition, an explanatory comment (`Disable automatic logon` ) adds context and helps users to identify the proposed configuration; the default one.

Reply:
I am afraid `<!-- Disable automatic logon. This is the default value. -->` is not sufficient and you need to say more like you have in your above response. You need to call out the security issue as a reason to explicitly add this default value.

Reply:
Makes senses, will adjust.